jolly-jellyfish/src/static/themes/console.css

```diff
@@ -13,6 +13,43 @@ body {
   font-family: 'Courier New', Courier, monospace;
 }
 
+img {
+  width: 30px;
+  height: 30px;
+  padding-left: 1ch;
+}
+
+.img-wrapper {
+  display: flex;
+  align-items: center;
+}
+
+.img-wrapper::before {
+  content: ">";
+}
+
+.img-wrapper:hover img {
+  width: auto;
+  height: auto;
+  max-width: 90vw;
+  max-height: 90vh;
+  margin: auto;
+  z-index: 10;
+}
+
+.img-wrapper:hover {
+  position: fixed;
+  height: 100%;
+  width: 100%;
+  top: 0;
+  z-index: 5;
+  background: #000d;
+}
+
+.img-wrapper:hover::before {
+  content: "<";
```

Comment:
not sure if I like this... maybe we could adjust the transparency (using some css transitions)?

Reply:
That exists as a button to close the modal

Reply:
Oh okay, nice.